packages/headless/src/database/mongodb/super-admin-role/super-admin.role.model.ts

```diff
@@ -0,0 +1,41 @@
+import { Role } from 'src/entity/role';
+import { Schema, model } from 'mongoose';
+import { randomUUID } from 'crypto';
+import { RoleTypeEnum } from 'models';
+
+const SuperAdminRoleSchema = new Schema<Role>(
+  {
+    id: {
+      type: String,
+      default: () => randomUUID(),
+      unique: true,
+      indexe: true,
```

Comment:
spelling mistake. and unique means unique index. no need to rewrite index

Reply:
Fix